WorkloadManagementSystem/Client/CPUNormalization.py

```diff
@@ -123,9 +124,12 @@ def getCPUNormalization( reference = 'HS06', iterations = 1 ):
   except ( TypeError, ValueError ), x :
     return S_ERROR( x )
 
-  # This number of iterations corresponds to 250 HS06 seconds
+  # This number of iterations corresponds to 1kHS2k.seconds, i.e. 250 HS06 seconds
+  # 06.11.2015: fixed absolute normalization w.r.t. MJF at GRIDKA
+  from DIRAC.ConfigurationSystem.Client.Helpers.Operations import Operations
+  corr = Operations().getValue( 'JobScheduling/CPUNormalizationCorrection', 1. )
```

Comment:
I am wondering if it wouldn't be better to just put 0.6 as default value instead of 1.0.